dependencies.yml

```diff
@@ -5,3 +5,4 @@ ZLIB_VERSION: 1.2.13
 # https://github.com/10gen/baas/commits
 # 2f308db is 2024 July 10
 BAAS_VERSION: 2f308db6f65333728a101d1fecbb792f9659a5ce
```

Comment:
is the server using v14 already? and more specifically, do we need to bump this version?

Reply:
Yes, the server has already been released - this is the same version that we are using on `master` (updated last week) which includes these changes.